src/main/scala/scorex/core/network/NetworkController.scala

```diff
@@ -31,11 +35,6 @@ class NetworkController(settings: NetworkSettings,
                         timeProvider: NetworkTimeProvider
                        ) extends Actor with ScorexLogging {
 
-  import NetworkController.ReceivableMessages._
-  import NetworkControllerSharedMessages.ReceivableMessages.DataFromPeer
-  import scorex.core.network.peer.PeerManager.ReceivableMessages.{CheckPeers, FilterPeers, Disconnected}
-  import PeerConnectionHandler.ReceivableMessages.CloseConnection
```

Comment:
@ceilican 
Should this imports be placed in the trait or moved to the top of the file like done here?

Reply:
I don't have a strong opinion about this. I just think that we should have a consistent style.

Reply:
I agree about consistent style. I'd better move them to the top